packages/design/src/Icon/index.js

```diff
@@ -14,333 +14,7 @@ See the License for the specific language governing permissions and
-  MoreVert,
-  Mute,
-  NewTab,
-  NoteAdded,
-  NotificationsActive,
-  OpenID,
-  Paypal,
-  PaperPlane,
-  Pencil,
-  Person,
-  PersonAdd,
-  PhonelinkErase,
-  PhonelinkSetup,
-  Planet,
-  Play,
-  PowerSwitch,
-  Profile,
-  Question,
-  Restore,
-  Server,
-  SettingsInputComposite,
-  SettingsOverscan,
-  Share,
-  Shart,
-  ShieldCheck,
-  Shrink,
-  SmallArrowDown,
-  SmallArrowUp,
-  Sort,
-  SortAsc,
-  SortDesc,
-  Speed,
-  Spinner,
-  Stars,
-  Stripe,
-  SyncAlt,
-  Tablet,
-  Tags,
-  Terminal,
-  Trash,
-  Twitter,
-  UsbDrive,
-  Unarchive,
-  Unlock,
-  Upload,
-  User,
-  UserCreated,
-  Users,
-  VideoGame,
-  Visa,
-  VolumeUp,
-  VpnKey,
-  Wand,
-  Warning,
-  Wifi,
-  Windows,
-  Youtube,
-} from './Icon';
+import Icon from './Icon';
 
 export default Icon;
-export {
-  Add,
-  AddUsers,
-  AmazonAws,
-  Amex,
-  Apartment,
-  AppInstalled,
-  Apple,
-  AppRollback,
-  Archive,
-  ArrowDown,
-  ArrowForward,
-  ArrowBack,
-  ArrowLeft,
-  ArrowRight,
-  ArrowsVertical,
-  ArrowUp,
-  AlarmRing,
-  BagDollar,
-  BitBucket,
-  Bubble,
-  Camera,
-  CardView,
-  CardViewSmall,
-  CaretLeft,
-  CaretRight,
-  CarrotDown,
-  CarrotLeft,
-  CarrotRight,
-  CarrotSort,
-  CarrotUp,
-  Cash,
-  ChevronCircleDown,
-  ChevronCircleLeft,
-  ChevronCircleRight,
-  ChevronCircleUp,
-  CircleArrowLeft,
-  CircleArrowRight,
-  CircleCheck,
-  CircleCross,
-  CirclePause,
-  CirclePlay,
-  CircleStop,
-  Cli,
-  Clipboard,
-  ClipboardUser,
-  Close,
-  Cloud,
-  CloudSync,
-  Cluster,
-  Clusters,
-  ClusterAdded,
-  ClusterAuth,
-  Code,
-  Cog,
-  Config,
-  Contract,
-  CreditCard,
-  CreditCardAlt,
-  CreditCardAlt2,
-  Cross,
-  Database,
-  Desktop,
-  Discover,
-  Download,
-  Earth,
-  Edit,
-  Ellipsis,
-  EmailSolid,
-  EnvelopeOpen,
-  Equalizer,
-  EqualizerVertical,
-  Expand,
-  Facebook,
-  FacebookSquare,
-  FileCode,
-  ForwarderAdded,
-  Github,
-  Google,
-  Graph,
-  Home,
-  Info,
-  Key,
-  Keypair,
-  Kubernetes,
-  Label,
-  Lan,
-  LanAlt,
-  Layers,
-  Layers1,
-  License,
-  Link,
-  Linkedin,
-  Linux,
-  List,
-  ListAddCheck,
-  ListBullet,
-  ListCheck,
-  ListThin,
-  ListView,
-  LocalPlay,
-  Lock,
-  Magnifier,
-  MasterCard,
-  Memory,
-  MoreHoriz,
-  MoreVert,
-  Mute,
-  NewTab,
-  NoteAdded,
-  NotificationsActive,
-  OpenID,
-  PaperPlane,
-  Paypal,
-  Pencil,
-  Person,
-  PersonAdd,
-  PhonelinkErase,
-  PhonelinkSetup,
-  Planet,
-  Play,
-  PowerSwitch,
-  Profile,
-  Question,
-  Restore,
-  Server,
-  SettingsInputComposite,
-  SettingsOverscan,
-  Share,
-  Shart,
-  ShieldCheck,
-  Shrink,
-  SmallArrowDown,
-  SmallArrowUp,
-  Sort,
-  SortAsc,
-  SortDesc,
-  Speed,
-  Spinner,
-  Stars,
-  Stripe,
-  SyncAlt,
-  Tablet,
-  Tags,
-  Terminal,
-  Trash,
-  Twitter,
-  UsbDrive,
-  Unarchive,
-  Unlock,
-  Upload,
-  User,
-  UserCreated,
-  Users,
-  VideoGame,
-  Visa,
-  VolumeUp,
-  VpnKey,
-  Wand,
-  Warning,
-  Wifi,
-  Windows,
-  Youtube,
-};
+export * from './Icon';
```

Comment:
Do you know why we used to export each icon individually? If not, could you ask one more person from the frontend team to review this change specifically?

The change makes sense to me, but the process of exporting those icons one by one seems elaborate enough that I suspect there must've been a reason behind it. Unfortunately, I don't see any documentation about it.

Reply:
@kimlisa maybe you know the reason for this? For me both versions work identically

Reply:
i don't know the history of why it was like this. if it doesn't break anything currently, i think it's okay?